Source/Tech Tree/Parts Browser/data/SmartParts.json

```diff
@@ -1,15 +1,15 @@
 [
     {
         "name": "FuelController",
-        "title": "",
-        "description": "",
+        "title": "KM Fuel Controller",
```

Comment:
This should not be included. Leave it in here, but change "orphan": true

Reply:
While not exactly the same thing, the Falcon Heavy was originally envisioned to have fuel crossfeed. You could use this to accomplish that, or even to shut down engines when you reach a specific fuel level (so that you could save some for a landing). 

I'll mark it as an orphan part for now, but just something to keep in mind.